src/main/java/chess/controller/ChessController.java

```diff
@@ -14,48 +18,77 @@
 public class ChessController {
     private final InputView inputView;
     private final OutputView outputView;
+    private final BoardRepository boardRepository;
 
-    public ChessController(final InputView inputView, final OutputView outputView) {
+    public ChessController(final InputView inputView, final OutputView outputView, BoardRepository boardRepository) {
         this.inputView = inputView;
         this.outputView = outputView;
+        this.boardRepository = boardRepository;
     }
 
     public void run() {
+        TableDao.createChessBoardIfNotExist();
         final GameCommand gameCommand = retryOnException(
-                () -> GameCommand.createFirstGameCommand(inputView.readGameCommand())
-        );
+                () -> GameCommand.createFirstGameCommand(inputView.readGameCommand()));
         if (gameCommand.isEnd()) {
             return;
         }
-        final ChessBoard chessBoard = new ChessBoard(ChessBoardInitializer.create());
+        final ChessBoard chessBoard = getChessBoard(boardRepository);
+        updateBoardChange(boardRepository, chessBoard);
         outputView.printChessBoard(chessBoard);
-        retryOnException(() -> playChess(chessBoard));
+        printPoints(retryOnException(() -> playChess(boardRepository)));
     }
 
-    private void playChess(final ChessBoard chessBoard) {
-        while (true) {
+    private ChessBoard getChessBoard(final BoardRepository boardRepository) {
+        if (boardRepository.count() != 0) {
+            return getAllDataFrom(boardRepository);
+        }
+        return new ChessBoard(ChessBoardInitializer.create());
+    }
+
+    private ChessBoard getAllDataFrom(final BoardRepository boardRepository) {
+        return boardRepository.findAll()
+                .convert();
+    }
+
+    private ChessBoard playChess(final BoardRepository boardRepository) {
+        ChessBoard chessBoard = getAllDataFrom(boardRepository);
+        while (!chessBoard.checkChessEnd()) {
             final GameArguments gameArguments = inputView.readGameArguments();
             final GameCommand gameCommand = gameArguments.gameCommand();
             if (gameCommand.isEnd()) {
                 break;
             }
+            if (gameCommand.isStatus()) {
+                printPoints(chessBoard);
+                continue;
+            }
             final MoveArguments moveArguments = gameArguments.moveArguments();
-            move(chessBoard, moveArguments);
+            chessBoard = move(chessBoard, moveArguments);
+            outputView.printChessBoard(chessBoard);
+            updateBoardChange(boardRepository, chessBoard);
         }
+        return chessBoard;
     }
 
-    private void move(final ChessBoard chessBoard, final MoveArguments moveArguments) {
+    private ChessBoard move(ChessBoard chessBoard, final MoveArguments moveArguments) {
```

Comment:
```suggestion
    private ChessBoard move(final ChessBoard chessBoard, final MoveArguments moveArguments) {
```
다온이 스스로 설정한 final 컨벤션이라면 누락된 곳이 있는지 잘 확인해주세요!